tests/unit/users/test_serializers.py

```diff
@@ -7,6 +7,7 @@
 from rest_framework.test import APIRequestFactory
 
 
+@patch('openslides.users.rest_auth.config', {'system_enable_anonymous': False})
```

Comment:
1. users.models.User.username: blank=True
2. users.serializers: remove __init__
3. Generate username in validate and not create method
4. Change Docstring
5. Try to merge FullUser and CreateUpdateUser (Test with a database where user is in group with pk 1 or 2)